components/molecules/chat/preview.vue

```diff
@@ -1,27 +1,31 @@
 <template>
-  <NuxtLink>
-    <AtomsCard
-      @mouseover="isButtonShow = true"
-      @mouseleave="isButtonShow = false"
-      @focus="() => {}"
-      @blur="() => {}"
+  <div
+    class="relative overflow-hidden rounded hover:bg-gray-600 hover:text-gray-200"
+  >
+    <NuxtLink
+      :to="`/chanels/${chat.chanelId}/${chat.id}`"
+      active-class="bg-gray-700 flex"
     >
-      <AtomsAvatarInitials
-        :avatar="anotherMember?.avatar"
-        :name="anotherMember?.name"
-        :aria-label="anotherMember?.name"
-      />
-      <p class="flex-1 truncate">
-        {{ chatName }}
-      </p>
-      <div
-        v-if="isButtonShow"
-        class="flex h-[16px] w-[16px] shrink-0 items-center"
-      >
-        <IconsClose />
-      </div>
-    </AtomsCard>
-  </NuxtLink>
+      <AtomsCard class="flex-1">
+        <AtomsAvatarInitials
+          :avatar="anotherMember?.avatar"
+          :name="anotherMember?.name"
+          :aria-label="anotherMember?.name"
+        />
+        <p class="flex-1 truncate">
+          {{ chatName }}
+        </p>
+      </AtomsCard>
+    </NuxtLink>
+
+    <button
+      type="button"
+      class="absolute top-1/2 right-5 flex h-4 w-4 shrink-0 -translate-y-1/2 items-center"
+      @click="($event) => $event.stopPropagation()"
+    >
+      <IconsClose class="fill-gray-400 hover:fill-gray-200" />
```

Comment:
maybe it's better to remove the button from dom

Reply:
a11y reasons. let it be, its rent-free